customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -0,0 +1,221 @@
+    },
+    "PluItemType": {
+      "description": "This is an entry for a specific item, and is placed inside a PLU list.",
+      "type": "object",
+      "required": [
+        "identifier",
+        "images",
+        "type"
+      ],
+      "properties": {
+        "images": {
+          "$ref": "#/$defs/ImagesType"
+        },
+        "identifier": {
+          "description": "The, item identifier reference",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 20
+        },
+        "type": {
+          "description": "The, ITEM type is for PLU items.",
+          "type": "string",
+          "enum": [
+            "ITEM"
+          ]
+        }
+      }
+    },
+    "PluGroupType": {
+      "description": "This is a PLU - it contains a set of other PLUs to form a hierarchy.",
+      "type": "object",
+      "required": [
+        "id",
+        "images",
+        "type"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 40,
+          "example": "Apples"
+        },
+        "description": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 255,
+          "example": "Our delicious apples"
+        },
+        "id": {
+          "type": "string"
+        },
+        "images": {
+          "$ref": "#/$defs/ImagesType"
+        },
+        "content": {
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/PluContentType"
+          }
+        },
+        "type": {
+          "description": "The GROUP type is for PLU lists.",
+          "type": "string",
+          "enum": [
+            "GROUP"
+          ]
+        }
+      }
+    },
+    "AdditionalProperty": {
+      "description": "Contains key-value collection of additional property.",
+      "type": "object",
+      "required": [
+        "id",
+        "value",
+        "dataType"
+      ],
+      "properties": {
+        "id": {
+          "description": "A, unique id for additional property object.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 255,
+          "example": "property.id"
+        },
+        "value": {
+          "description": "The actual Value of the additional property object.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 255,
+          "example": "50.00"
+        },
+        "dataType": {
+          "description": "The type of the data found in 'value'.",
+          "type": "string",
+          "example": "DOUBLE",
+          "enum": [
+            "BOOL",
+            "INT",
+            "DOUBLE",
+            "STRING",
+            "DATE",
+            "DATETIME",
+            "IMAGE",
+            "BINARY",
+            "JSON",
+            "ARRAY",
+            "UNKNOWN"
+          ]
+        }
+      }
+    },
+    "ImagesType": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
```

Comment:
the `url` property should be required for the image, but the images may be an optional field, giving us more control and flexibility.
I think consumers would expect to have a valid URL of the image, if it's in the schema